src/status_im/transport/db.cljs

```diff
@@ -0,0 +1,29 @@
+(ns status-im.transport.db
+  (:require-macros [status-im.utils.db :refer [allowed-keys]])
+  (:require [cljs.spec.alpha :as spec]))
+
+;; required
+(spec/def ::ack (spec/coll-of string? :kind vector?))
+(spec/def ::seen (spec/coll-of string? :kind vector?))
+(spec/def ::pending-ack (spec/coll-of string? :kind vector?))
+(spec/def ::pending-send (spec/coll-of string? :kind vector?))
+(spec/def ::topic string?)
+
+;; optional
+(spec/def ::sym-key-id string?)
+;;TODO (yenda) remove once go implements persistence
+(spec/def ::sym-key string?)
+(spec/def ::filter any?)
+
+(spec/def :transport/chat (allowed-keys :req-un [::ack ::seen ::pending-ack ::pending-send ::topic]
+                                        :opt-un [::sym-key-id ::sym-key ::filter]))
+
+(spec/def :transport/chats (spec/map-of :global/not-empty-string :transport/chat))
+(spec/def :transport/discovery-filter (spec/nilable any?))
+
+(defn create-chat [topic]
+  {:ack          []
+   :seen         []
+   :pending-ack  []
+   :pending-send []
+   :topic        topic})
```

Comment:
I don't understand this abstraction. Why does a "chat" have these 5 things in it but not e.g. messages or sym-key (since topic is in it)?

Reply:
I think this is the representation of a chat at the transport layer `:transport/chats` as opposed to `:chats/:chat` i.e. the one with messages/sym-key etc. 
It looks like those are in `status-im.chat.specs`.

Reply:
@cammellos no this is where the sym-key will be as well but there is not sym-key yet when this is created, the sym-key is added when it is created. This is just a helper function to get the initial values.